package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "codeo-headless-demo-080",
+  "name": "coveo-headless-demo0150",
   "version": "1.0.0",
   "description": "",
   "keywords": [],
   "main": "src/index.tsx",
   "dependencies": {
-    "@coveo/headless": "0.10.0",
```

Comment:
hmmm... was this branched off of an old sandbox?
This could be causing update issues. For instance, it might explain why the relevance inspector does not appear: 

<img width="1339" alt="Screen Shot 2021-06-01 at 3 04 00 PM" src="https://user-images.githubusercontent.com/39384459/120377048-9b62a080-c2ea-11eb-8f51-be80c916dad9.png">

Versus the prod sandbox on the latest branch:

<img width="876" alt="Screen Shot 2021-06-01 at 3 04 55 PM" src="https://user-images.githubusercontent.com/39384459/120377146-bcc38c80-c2ea-11eb-8c44-e6dde03b78ea.png">

Reply:
this is really odd. I continue to have this issue, despite changing it in CSB. IT must be that the UI version toggle is not writing to the package.json file or something.

Reply:
Oh, right the debug toggle is now just under the query summary. I noticed this is how it was placed in the repo sent to us. My instinct was to put it on the bottom (which I did on my first implementation). After reflecting on that choice, in the context of presenting a feature, I realized it shouldn't be hidden down on the bottom of the page.